client/app/reducers/token.js

```diff
@@ -0,0 +1,11 @@
+import { TOKEN } from '../constants/token'
+const initialState = {}
```

Comment:
this is were you can define your token as false, or maybe uninitialized: true or something along those lines.

Reply:
I'm also missing the usage of immutable or seamless-immutable